emsdk.py

```diff
@@ -2637,11 +2637,14 @@ def error_on_missing_tool(name):
 
 
 def expand_sdk_name(name):
+  if name in ('latest-fastcomp', 'latest-releases-fastcomp'):
+    # Since we no longer support fastcomp in ToT emscripten, the latest
+    # fastcomp release is fixed at 1.40.1.
+    name = 'sdk-fastcomp-1.40.1'
```

Comment:
I'm actually not sure this is what we want.

This way people silently get `1.40.1` for `latest-fastcomp` forever, instead of an error suggesting they get upstream.

Not urgent in any way, but let's think about this.

Reply:
I was thinking that is the latest fastcomp release... but maybe its better to fail if people as for it that way?